pkg/network/kubemacpool_test.go

```diff
@@ -0,0 +1,102 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+var _ = Describe("kubemacpool_test", func() {
+	It("should NOT return an error because kube mac pool can be nil", func() {
```

Comment:
Someone please correct me if I'm wrong, but I think that in Ginkgo, this should be something like:

```go
    Context("When kubeMacPool is set to nil", func() {
		It("validation should not fail", func() {
			clusterConfig := &opv1alpha1.NetworkAddonsConfigSpec{KubeMacPool: &opv1alpha1.KubeMacPool{}}
			errorList := validateKubeMacPool(clusterConfig)
			Expect(errorList).To(BeEmpty())
		})
	})
```

The same for the others, don't hesitate to ping me if you stuck somewhere.